collect_app/src/main/java/org/odk/collect/android/formmanagement/formmap/FormMapViewModel.kt

```diff
@@ -163,7 +163,7 @@ class FormMapViewModel(
     private fun instanceStatusToMappableSelectionItemStatus(instance: Instance): Status? {
         return when (instance.status) {
             Instance.STATUS_INVALID, Instance.STATUS_INCOMPLETE -> Status.ERRORS
-            Instance.STATUS_VALID -> Status.NO_ERRORS
+            Instance.STATUS_VALID, Instance.STATUS_NEW_EDIT -> Status.NO_ERRORS
```

Comment:
I don't think there's a test for this change

Reply:
Done.